src/Surfnet/StepupSelfService/SelfServiceBundle/Service/RemoteVetting/Value/AttributeMatch.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright 2020 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupSelfService\SelfServiceBundle\Service\RemoteVetting\Value;
+
+class AttributeMatch
```

Comment:
It's used in a form to fill the RemoteVetValidationCommand with a collection of AttributeMatch VO's. The problem is that Symfony does not use DI for form data objects by default. I agree it would be much better to add a DataMapper to do this so the object can become immutable. I will add this as a task.

Reply:
I'm totaly fine to let Symfony do it's magic on the object. But storing it in a Value Object namespace is simply not right. But addressing that when the aforementioned task is picked up is fine with me.